src/app/home-page/widget/organization-box/organization-box.component.html

```diff
@@ -0,0 +1,135 @@
+              in organization
+              <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+              <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+            </div>
+            <div *ngSwitchCase="EventType.ADDTOCOLLECTION" class="my-3">
+              <strong>{{ event.initiatorUser.username }} </strong> added the {{ event?.tool || event?.apptool ? 'tool' : 'workflow' }}
+              <a
+                [routerLink]="
+                  event?.apptool
+                    ? '/tools/' + event?.apptool.full_workflow_path
+                    : event?.tool
+                    ? '/tools/' + event?.tool.tool_path
+                    : '/workflows/' + event?.workflow.full_workflow_path
+                "
+                >{{ event?.apptool ? event?.apptool.workflowName : event?.tool ? event?.tool.name : event?.workflow.workflowName }}</a
+              >
+              to the collection
+              <a [routerLink]="'/organizations/' + event.organization?.name + '/collections/' + event.collection?.name"
+                >{{ event.collection?.name }}</a>
+              in organization
+              <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+              <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+            </div>
+            <div *ngSwitchCase="EventType.ADDUSERTOORG" class="my-3">
+              <strong>{{ event.initiatorUser.username }} </strong> added {{ event.user.username }} to the organization
+              <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+              <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+            </div>
+            <div *ngSwitchCase="EventType.MODIFYCOLLECTION" class="my-3">
+              <strong>{{ event.initiatorUser.username }} </strong> edited the collection
+              <a [routerLink]="'/organizations/' + event.organization?.name + '/collections/' + event.collection?.name"
+                >{{ event.collection?.name }}</a>
+              in organization
+              <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+              <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+            </div>
+            <div *ngSwitchCase="EventType.MODIFYORG" class="my-3">
+              <strong>{{ event.initiatorUser.username }} </strong> edited the organization
+              <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+              <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+            </div>
+          </div> -->
+    <!-- </div> -->
+    <!-- <img class="mx-2" src="{{ event?.organization.avatarUrl }}" alt="Org avatar" />
+      </div> -->
```

Comment:
commented this part out before we decide if we want to move forward with listing organization names rather than events.